packages/nbdime/src/diff/widget/output.ts

```diff
@@ -422,7 +421,7 @@ class RenderableOutputView extends RenderableDiffView<nbformat.IOutput> {
     let i = 0;
     let model = this.model;
     this.mimetype = mimeType;
-    each(this.layout.widgets, (w: Widget) => {
+    each(this.layout!.widgets, (w: Widget) => {
```

Comment:
It should be possible to drop `each`, `find` and `toArray` in this file rewriting to native JS. We could then drop `@lumino/algorithm` here. This can be done after this PR is merged.

Reply:
I would suggest adding this to PR description as a follow up task. I don't think this is worth changing now, there are more important things to address.

Reply:
Ok. This has been added to the section "Future work and follow up tasks" of the description of this PR.